packages/did-provider-oyd/src/oyd-did-provider.ts

```diff
@@ -48,7 +48,7 @@ export class OydDIDProvider extends AbstractIdentifierProvider {
     const keyType: OydDidSupportedKeyTypes = options?.keyType || 'Ed25519'
     const key = await this.holdKeys(
       {
-        kms: kms || this.defaultKms,
+        kms: kms || this.defaultKms || '',
```

Comment:
Rather have a ts-ignore here. Our KMS handles it correctly when no KMS is provided. Using an empty string will result in errors